Test/test_system.py

```diff
@@ -0,0 +1,51 @@
+import subprocess
+import unittest
+import time
+import os
+import Client.python_client
+import Client.status
+
+
+class MyTestCase(unittest.TestCase):
+    def test_system(self):
+        """
+        This function tests the system
+        """
+
+        self.file_path = "../asyncio-intro.pptx"
+
+        self.scripts = ['../Web_api/app.py', '../Gpt_explainer/main.py']
+
+        self.processes = []
+
+        for script in self.scripts:
+            process = subprocess.Popen(['python', script])
+            self.processes.append(process)
+
+        my_client = Client.python_client.PythonClient()
+
+        try:
+            uid = my_client.send_file(os.path.abspath(self.file_path))
```

Comment:
Where are the assertions? You should verify that everything works
https://www.w3schools.com/python/ref_keyword_assert.asp